src/services/refactors/convertExport.ts

```diff
@@ -122,7 +123,7 @@ function getInfo(context: RefactorContext, considerPartialSpans = true): ExportI
     }
 
     const checker = program.getTypeChecker();
-    const exportingModuleSymbol = getExportingModuleSymbol(exportNode, checker);
+    const exportingModuleSymbol = getExportingModuleSymbol(exportNode.parent, checker);

```

Comment:
what changed, the node where the symbol is bound?

Reply:
Line 323: https://github.com/microsoft/TypeScript/pull/51682/files/70cdafcd12690c95d1201588b3483a7e2921cf1c?show-viewed-files=true&file-filters%5B%5D=#diff-452063e7824422df714dad139815ed8b72fe1f4d301905302087cb37d9ca416cR323

This just avoids an unsound cast in `getExportingModuleSymbol`, since `Node` doesn't have a `symbol` property any more now that it was moved to `Declaration`.